docs/quick-start.rst

```diff
@@ -3,12 +3,21 @@ SNARE
 
 Super Next generation Advanced Reactive honEypot
 
+Snare is a web application honeypot and is the successor of Glastopf, which has many of the same
+features as Glastopf as well as ability to convert existing Web pages into attack surfaces with TANNER_.
```

Comment:
which features? Please do not assume that all people are familiar with Glastopf features, you may either give a link to Glastopf features or write the list of supported features.

Reply:
A link to the Glastopf repository would suffice? as we have features of Glastopf mentioned in its Readme.

Reply:
Yes, should be sufficient :)